winlogbeat/docs/fields.asciidoc

```diff
@@ -215,7 +215,7 @@ Contains common fields available in all event types.
 required: True
 
 The event log API type used to read the record. The possible values are "wineventlog" for the Windows Event Log API or "eventlogging" for the Event Logging API.
-The Event Logging API was designed for Windows Server 2003 or Windows 2000 operating systems. In Windows Vista, the event logging infrastructure was redesigned. On Windows Vista or later operating systems, the Windows Event Log API is used. Winlogbeat automatically detects which API to use for reading event logs.
+The Event Logging API was designed for Windows Server 2003, Windows XP, or Windows 2000 operating systems. In Windows Vista, the event logging infrastructure was redesigned. On Windows Vista or later operating systems, the Windows Event Log API is used. Winlogbeat automatically detects which API to use for reading event logs.
```

Comment:
I wonder how that got in again as I think @dedemorton removed it recently.

Reply:
Indeed I did: https://github.com/elastic/beats/commit/33b4f9a0a339f8cbc7893ddb93b53017bef5fb2a#diff-5ef49766a1ba7710815be5733d346e9e